CFC_DataCollector/modeinfer/pipeline.py

```diff
@@ -83,20 +98,42 @@ def getSectionQueryWithGroundTruth(groundTruthMode):
 
   # TODO: Refactor into generic steps and results
   def loadTrainingDataStep(self, sectionQuery, sectionDb = None):
+    logging.debug("START TRAINING DATA STEP")
     if (sectionDb == None):
       sectionDb = self.Sections
+    begin = time.time()
     logging.debug("Section data set size = %s" % sectionDb.find({'type': 'move'}).count())
+    duration = time.time() - begin
+    logging.debug("Getting dataset size took %s" % (duration))
+        
+    logging.debug("Querying confirmedSections %s" % (datetime.datetime.now()))
```

Comment:
Please don't put the datetimes at the end of the log statements. The log statements are already timestamped, so this is a bunch of unnecessary system calls. It is true that the calls are pretty lightweight, but why do more work than we need to. if your log statements don't print the time, you can look into the log format that is getting invoked.